wandb/sdk/launch/runner/sagemaker_runner.py

```diff
@@ -229,7 +244,10 @@ def run(
 
 
 def merge_aws_tag_with_algorithm_specification(
-    algorithm_specification: Optional[Dict[str, Any]], aws_tag: Optional[str]
+    algorithm_specification: Optional[Dict[str, Any]],
+    aws_tag: Optional[str],
```

Comment:
Not sure about calling this aws_tag, it's an image uri right? I think you can run public dockerhub images on sagemaker?  I think argument descriptions in the docstring would be helpful here